fe/fe-core/src/main/java/org/apache/doris/datasource/property/storage/S3Properties.java

```diff
@@ -280,8 +282,7 @@ private void convertGlueToS3EndpointIfNeeded() {
         }
     }
 
-    @Override
-    public AwsCredentialsProvider getAwsCredentialsProvider() {
```

Comment:
it is better we do the provider inside this function instead removing this function and add other new names xxv1 xxv2.
e.g.
```
    public AwsCredentialsProvider getAwsCredentialsProvider() {
       if (config.v1) return providerChainV1()
       else (config.v2) return providerChainV2()
}

```